### ISSUE TITLE
Ingest API 404 for JFR batch

### ISSUE THREAD

**Issue by kkocel** [newrelic/newrelic-jfr-core#64]
[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )

## Description

JFR data batches cannot be sent - 404 error endpoint

## Relevant Logs / Console output
```
2020-08-07T08:53:11,008+0000 [1 16] com.newrelic WARN: Response from New Relic ingest API. Discarding batch recommended.: code: 404, body: {}
2020-08-07T08:53:11,009+0000 [1 16] com.newrelic ERROR: Received a fatal exception from the New Relic API. Aborting metric batch send.
com.newrelic.deps.jfr.telemetry.exceptions.DiscardBatchException: The New Relic API failed to process this request and it should not be retried.
	at com.newrelic.deps.jfr.telemetry.transport.BatchDataSender.sendPayload(BatchDataSender.java:134) ~[?:?]
	at com.newrelic.deps.jfr.telemetry.transport.BatchDataSender.send(BatchDataSender.java:81) ~[?:?]
	at com.newrelic.deps.jfr.telemetry.metrics.MetricBatchSender.sendBatch(MetricBatchSender.java:67) ~[?:?]
	at com.newrelic.deps.jfr.telemetry.TelemetryClient.lambda$sendBatch$0(TelemetryClient.java:122) ~[?:?]
	at com.newrelic.deps.jfr.telemetry.TelemetryClient.sendWithErrorHandling(TelemetryClient.java:173) ~[?:?]
	at com.newrelic.deps.jfr.telemetry.TelemetryClient.lambda$scheduleBatchSend$4(TelemetryClient.java:167) ~[?:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:?]
	at java.util.concurrent.FutureTask.run(Unknown Source) [?:?]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [?:?]
	at java.lang.Thread.run(Unknown Source) [?:?]
2020-08-07T08:53:12,437+0000 [1 16] com.newrelic WARN: Response from New Relic ingest API. Discarding batch recommended.: code: 404, body: {}
```
## Your Environment

New Relic Agent v5.14.0
jfr-reporter-0.2.0

**Comment by breedx-nr** [comment]
Hi @kkocel.  Thanks for this bug report.  Two notes:
1. This is great info, I experienced the same behavior on an unrelated test last night, so consider this confirmed.
2. Ideally this would have been filed against the `jfr-reporter` repository, but we don't yet have that on github.com yet (expect it soon)...so I completely understand.

As a workaround, can you please try putting this full URI config setting into your `newrelic.yml` file under the `common` block and let us know if that helps?
```
  metric_ingest_uri: 'https://metric-api.newrelic.com/metric/v1'
```
or, if you are reporting to the ingest endpoints in the EU:
```
  metric_ingest_uri: 'https://metric-api.eu.newrelic.com/metric/v1'
```

Even if that does get you unblocked, I want to keep this open until we can either migrate this issue to the other repo (once we finish getting it open) or fix it in the other repo.  

Thanks again!

**Comment by kkocel** [comment]
Hi, I confirm that setting ingest URI (first one) resolved my issue - now should I wait for a permanent fix?

**Comment by breedx-nr** [comment]
@kkocel Yes, thanks for the confirmation that the workaround helped!  Yes, we should fix this in the reporter repo.  It's probably just a matter of fixing the default URL value.

**Comment by breedx-nr** [comment]
Hi @kkocel .

We released [jfr-reporter 0.3.0](https://oss.sonatype.org/content/groups/public/com/newrelic/agent/extension/jfr-reporter/0.3.0/) today.

Hoping that you might have some time to try it out without the above workaround and let us know.  Thanks!  🥂
